Source/WebInspectorUI/UserInterface/Views/Popover.js

```diff
@@ -598,6 +599,7 @@ WI.Popover = class Popover extends WI.Object
             window.addEventListener("scroll", this, true);
             window.addEventListener("resize", this, true);
             window.addEventListener("keypress", this, true);
+            WI.tabBrowser.addEventListener(WI.TabBrowser.Event.SelectedTabContentViewDidChange, this.dismiss, this);
```

Comment:
so this is just specific to a tab changing.  what about if the current `WI.ContentView` with the same tab changes?  that's why id suggested `WI.ContentBrowser.Event.CurrentContentViewDidChange` or `WI.ContentViewContainer.Event.CurrentContentViewDidChange` because i think those will be even more broad and therefore probably be more safe (tho we'd probably want to verify that no `WI.Popover` creates a `WI.ContentBrowser` or `WI.ContentViewContainer` just in case)